contentcuration/contentcuration/frontend/shared/views/__tests__/studioBanner.spec.js

```diff
@@ -0,0 +1,54 @@
+import { render, screen } from '@testing-library/vue';
+import VueRouter from 'vue-router';
+import StudioBanner from '../StudioBanner.vue';
+
+// Mock the Kolibri design system composable
+const mockSendPoliteMessage = jest.fn();
```

Comment:
Here's mock of `useKLiveRegion`, but I don't see it used anymore for the banner.